src/PrestaShopBundle/Resources/config/services/core/grid/grid_data_factory.yml

```diff
@@ -86,6 +86,19 @@ services:
             - "@prestashop.core.localization.locale.context_locale"
             - "@=service('prestashop.adapter.legacy.context').getContext().currency.iso_code"
 
+    prestashop.core.grid.data_provider.email_logs:
```

Comment:
This looks like a copy of existing service `prestashop.core.grid.data_factory.email_logs` Why did you need to duplicate it?
I think you can keep/use the original and define your decorator along with it in `src/PrestaShopBundle/Resources/config/services/core/grid/grid_data_factory.yml`